admin/src/app/home/orders/stock-orders/receive/services/indexdb.ts

```diff
@@ -0,0 +1,33 @@
+import Dexie from "dexie";
+
+// Declare Database
+class ProductDatabase extends Dexie {
+  products: Dexie.Table<any, number>;
+
+  constructor() {
+    super("ProductDatabase");
+
+    this.version(1).stores({
+      products:
+        "id, productModelId, productModelName, productModelSku, orderQuantity, fulfilledQuantity, receivedQuantity",
+    });
+  }
+}
+
+class APIQueueDatabase extends Dexie {
+  skus: Dexie.Table<any, number>;
+
+  constructor() {
+    super("APIQueueDatabase");
+
+    this.version(1).stores({
+      skus: "++id, sku",
```

Comment:
table name `skus` doesn't make sense

Reply:
It'll only hold skus in rows to call the APIs